querybook/webapp/ui/SimpleReactSelect/SimpleReactSelect.tsx

```diff
@@ -36,6 +36,8 @@ export function SimpleReactSelect<T>({
     selectProps = {},
     withDeselect = false,
     clearAfterSelect = false,
+    optionSelector = (val) => val?.value,
+    ...otherParams
```

Comment:
there is no otherParams here from ISimpleReactSelectProps? If you are trying to pass props to the Select component, you need to add it to the interface